web/client/components/resources/forms/Metadata.jsx

```diff
@@ -88,16 +88,21 @@ class Metadata extends React.Component {
                     disabled={this.props.resource.saving}
                     placeholder={this.props.descriptionPlaceholderText}
                     defaultValue={this.props.resource ? this.props.resource.description : ""}
-                    value={this.props.resource && this.props.resource.metadata && this.props.resource.metadata.description || ""}/>
-            </FormGroup>
-            <FormGroup>
-                <ControlLabel>{this.props.createdAtFieldText}</ControlLabel>
-                <ControlLabel>{this.props.resource && this.renderDate(this.props.resource.createdAt) || ""}</ControlLabel>
-            </FormGroup>
-            <FormGroup>
-                <ControlLabel>{this.props.modifiedAtFieldText}</ControlLabel>
-                <ControlLabel>{this.props.resource && this.renderDate(this.props.resource.modifiedAt || this.props.resource.createdAt) || ""}</ControlLabel>
+                    value={this.props.resource && this.props.resource.metadata && this.props.resource.metadata.description || ""} />
             </FormGroup>
+            {
+                this.props.resource && this.props.resource.createdAt && <FormGroup>
+                    <ControlLabel>{this.props.createdAtFieldText}</ControlLabel>
+                    <ControlLabel>{this.props.resource && this.renderDate(this.props.resource.createdAt) || ""}</ControlLabel>
+                </FormGroup>
+            }
+            {
+                this.props.resource && this.props.resource.createdAt && <FormGroup>
+                    <ControlLabel>{this.props.modifiedAtFieldText}</ControlLabel>
+                    <ControlLabel>{this.props.resource && this.renderDate(this.props.resource.modifiedAt || this.props.resource.createdAt) || ""}</ControlLabel>
```

Comment:
@offtherailz do we show created when modified is not available?

with your response we will adapt the condition to show this field, which actually is wrong in every case

Reply:
Of course not, I think is a copy and paste error.

Reply:
That's correct.

Reply:
sorry, my fault... it somehow correct, the last mod in this case is the creation date.

Reply:
> Of course not, I think is a copy and paste error.

We agreed last time that when modifiedAt is not available then I should show createdAt

Reply:
Yes, I noticed one second 2 minutes after my comment that I was wrong